### DISCUSSION TITLE
Headless client

### DISCUSSION THREAD

**Discussion by VComrade** [Foxlider/FASTER#136] [Q&A]
Hello again, so I managed to get server working, but I have a question.
If I want to use another PC that is connected to my server as a headless client what setup is needed on the other PC to use it as a headless client?

**Comment by jupster** [comment]
I havent done this and am way tired to look this up but... I am not sure if it needs arma or not. It needs the mods you tell it. Dont use faster for the HC laptop use just the vanilla way of connecting. Someone elwe may have more knowledge but that is the basic info
